### ISSUE TITLE
RSS HRV channel import problem

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/meteosatlib#1]
(as opened by Maciej Filocha on sf mailinglist)

This command works as expected:

    gdalwarp -t_srs "+proj=latlong" -te 14 48 25 55 -of GTiff 
    H:MSG1_RSS:VIS006:201509230700 VIS006_201509230700.tiff

but that gives empty (no-data) output:

    gdalwarp -overwrite -t_srs "+proj=latlong" -te 14 48 25 55 -of GTiff 
    H:MSG1_RSS:HRV:201509230700 HRV_201509230700.tiff

However, normal (non-RSS) distribution channel works as a charm:

    gdalwarp -overwrite -t_srs "+proj=latlong" -te 14 48 25 55 -of GTiff 
    H:MSG3:HRV:201509230700 HRV_201509230700.tiff

Strace output from second command shows that no attempt is made to open 
input files, except the prologue and epilogue ones (and the usual scan 
of all files with matching timestamp).

using version 1.4, build from svn sources.

**Comment by mfilocha** [comment]
Sample data available here:
http://geo.nebula.grid.icm.edu.pl/eumetsat/pliszka/

H-000-MSG1__-MSG1_RSS____-_________-EPI______-201509230700-__
H-000-MSG1__-MSG1_RSS____-HRV______-000022___-201509230700-C_
H-000-MSG1__-MSG1_RSS____-HRV______-000023___-201509230700-C_
H-000-MSG1__-MSG1_RSS____-_________-PRO______-201509230700-__
H-000-MSG1__-MSG1_RSS____-VIS006___-000008___-201509230700-C_
H-000-MSG1__-MSG1_RSS____-VIS008___-000008___-201509230700-C_

**Comment by brancomat** [comment]
Still kind of confirmed in e4383583e40ba632e3d6c7e7e4495aeab84bf755, i.e.:

this works:
```
$ msat --display H:MSG2_RSS:WV_073:201604281255
```

this doesn't:
```
$ msat --display H:MSG2_RSS:HRV:201604281255
Floating point exception
```

**Comment by spanezz** [comment]
In the RSS branch, the RSS-HRV images are now visible, although I still have not checked if georeferencing is ok.

**Comment by spanezz** [comment]
Georeferencing is ok in non-HRV RSS, and currently not ok in HRV RSS. Now trying to fix it...

**Comment by spanezz** [comment]
Fixed in 91a0732a7d97cfba3dba8e9c68d958627ac00d79.
RSS (both HRV and non-HRV) should work fine now.
